monkey/common/utils/file_utils.py

```diff
@@ -1,5 +1,11 @@
 import os
 
 
+class InvalidPath(Exception):
+    pass
+
+
```

Comment:
Why isn't this in `common/utils/exceptions.py`?

Reply:
On some discussion, @mssalvatore pointed out that it's probably not a good idea to put all exceptions into a single file and I agree. This makes `file_utils.py` more self-contained and the exception is more explicit.